internal/cmprefimpl/cmppg/fuzz_test.go

```diff
@@ -40,7 +40,7 @@ func TestFuzz(t *testing.T) {
 				t.Skip("Causes unmarshalling to fail for derivative geometry")
 			}
 
-			want, err := BatchPostGIS{pg.db}.Unary(g)
+			want, err := BatchPostGIS(pg).Unary(g)
```

Comment:
Does this change compile? I haven't run it through the full CI, but when I manually make this change to my local I get: `cannot convert pg.db (type *sql.DB) to type BatchPostGIS`.

Reply:
You're right 👍